internal/statemachine/classic_states.go

```diff
@@ -92,3 +92,20 @@ func (stateMachine *StateMachine) runLiveBuild() error {
 
 	return nil
 }
+
+// Generate the manifest
+func (stateMachine *StateMachine) generatePackageManifest() error {
+	// This is basically just a wrapper around dpkg-query
+
+	outputPath := filepath.Join(stateMachine.commonFlags.OutputDir, "filesystem.manifest")
+	cmd := execCommand("sudo", "chroot", stateMachine.tempDirs.rootfs, "dpkg-query", "-W", "--showformat=${Package} ${Version}\n")
+	manifest, err := os.Create(outputPath)
+	if err != nil {
+		return fmt.Errorf("Error creating manifest file: %s", err.Error())
+	}
+	defer manifest.Close()
+
+	cmd.Stdout = manifest
```

Comment:
Setting the Stdout of the command to be the output file itself... this is clever! I like this.